tests/BootstrapTest.php

```diff
@@ -48,8 +50,63 @@ public function testWorkingFromChildWorkingDirectory()
         $this->assertStringEndsNotWith("/tests", getcwd());
         $this->assertStringStartsNotWith(".././", getenv("LOTGD_CONFIG"));
     }
+    
+    public function testBootstrapLoadsPackageModels()
+    {
+        $installationManager = $this->getMockBuilder(InstallationManager::class)
+            ->disableOriginalConstructor()
+            ->setMethods(["getInstallPath"])
+            ->getMock();
+        $installationManager->method("getInstallPath")->willReturn(__DIR__ . "/FakeModule");
+        
+        $composer = $this->getMockBuilder(\Composer\Composer::class)
+            ->disableOriginalConstructor()
+            ->setMethods(["getInstallationManager", "translateNamespaceToPath"])
+            ->getMock();
+        $composer->method("getInstallationManager")->willReturn($installationManager);
+        $composer
+            ->expects($this->exactly(1))
+            ->method("translateNamespaceToPath")
+            ->with("LotGD\\Core\\Tests\\FakeModule\\Models")
+            ->willReturn(__DIR__ . "/FakeModule/Models");
+        
+        $fakeModulePackage = $this->getMockBuilder(AliasPackage::class)
+            ->disableOriginalConstructor()
+            ->setMethods(["getType", "getExtra"])
+            ->getMock();
+        $fakeModulePackage->method("getType")->willReturn("lotgd-module");
+        $fakeModulePackage->method("getExtra")->willReturn(["lotgd-namespace" => "LotGD\\Core\\Tests\\FakeModule\\"]);
+        
+        $composerManager = $this->getMockBuilder(ComposerManager::class)
+            ->setMethods(["getPackages", "getComposer"])
+            ->getMock();
+        $composerManager->method("getPackages")->willReturn([$fakeModulePackage]);
+        $composerManager->method("getComposer")->willReturn($composer);
+        
+        $bootstrap = $this->getMockBuilder(Bootstrap::class)
+            ->setMethods(["createComposerManager"])
+            ->getMock();
+        $bootstrap->method("createComposerManager")->willReturn($composerManager);
+        
+        // run tests
+        $game = $bootstrap->getGame();
+        
+        $this->assertGreaterThanorEqual(3, $bootstrap->getReadAnnotationDirectories());
+        
+        /*$user = new UserEntity();
```

Comment:
Let's not leave any comments in the code. #cleancode

We can always get this code back in the git history.